variables.tf

```diff
@@ -2,8 +2,60 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "resource_group_id" {
+  description = "The Id of an existing IBM Cloud resource group where the watsonx Assistant instance will be grouped."
+  type        = string
+}
+
+variable "region" {
+  description = "IBM Cloud region where watsonx Assistant instance will be created."
+  type        = string
```

Comment:
same as above comment - region is only required if creating a new instance

Reply:
done